### ISSUE TITLE
[BUG] Convert to HTML works for one scene, but does not work for multiple scenes

### ISSUE THREAD

**Issue by mcnica89** [jeertmans/manim-slides#503]
### Terms

- [X] Checked the [existing issues](https://github.com/jeertmans/manim-slides/issues?q=is%3Aissue+label%3Abug+) and [discussions](https://github.com/jeertmans/manim-slides/discussions) to see if my issue had not already been reported;
- [X] Checked the [frequently asked questions]](https://manim-slides.eertmans.be/latest/faq.html);
- [X] Read the [installation instructions](https://manim-slides.eertmans.be/latest/installation.html);
- [ ] Created a virtual environment in which I can reproduce my bug;

### Describe the issue

When I use the convert to html feature with multiple scenes, the folder with all the .mp4 media is created correctly,. but the html file that plays from it is not working for some reason. It is simply black and I cannot advance slides (I can use "F" to make it fullscreen). When I do one scene at a time, it DOES work correctly. This issue was also reported on reddit https://www.reddit.com/r/manim/comments/1fzcv7z/manimslides_convert_gives_html_that_is_a_blank/ (which is what made me try rendering just one scene at a time)

### Command

manim-slides convert Scene1 Scene2 output.html

### Issue Type

Bug when presenting with HTML/PowerPoint/... format (`manim-slides convert`)

### Python version

Python 3.9.12

### Python environment

```shell
Manim Slides version: 5.1.10
Python executable: /Users/mnica/opt/anaconda3/bin/python3
Manim bindings:
/Users/mnica/opt/anaconda3/lib/python3.9/site-packages/scipy/__init__.py:146: UserWarning: A NumPy version >=1.16.5 and <1.23.0 is required for this version of SciPy (detected version 1.26.4
  warnings.warn(f"A NumPy version >={np_minversion} and <{np_maxversion}"
	manim (version: 0.18.1)
	manimgl not found
Qt API: pyqt5 (version: 5.9.7)
(base) mnica@Mihais-MBP Downloads % manim-slides checkhealth
Manim Slides version: 5.1.10
Python executable: /Users/mnica/opt/anaconda3/bin/python3
Manim bindings:
/Users/mnica/opt/anaconda3/lib/python3.9/site-packages/scipy/__init__.py:146: UserWarning: A NumPy version >=1.16.5 and <1.23.0 is required for this version of SciPy (detected version 1.26.4
  warnings.warn(f"A NumPy version >={np_minversion} and <{np_maxversion}"
	manim (version: 0.18.1)
	manimgl not found
Qt API: pyqt5 (version: 5.9.7)
```

### What is your platform?

macOS

**Comment by jeertmans** [comment]
Hi @mcnica89, thanks for reporting this bug!

I feel this is a duplicate of #496, did you check it?

If that is the case, a fix has already been merged, but was not yet released to PyPI. I just released `manim-slides==5.2.0`, which should contain the fix.

Let me know :-)

**Comment by svandenhaute** [comment]
I am encountering the same issue on the new 5.3.0. Multiple scenes are rendered and presented correctly using `manim-slides present`, but when converting to HTML it simply shows a black screen after the first scene.

**Comment by MANISHJOSHI23** [comment]
I am also facing the same issue on manim-slides 5.3.0. When I use default configuration without using --use-template the prefix is added to the html file. But when I copied the same configuration file and used  use-template command the prefix is not added to the html file

**Comment by jeertmans** [comment]
@svandenhaute, could you provide a minimal working example (MWE)? I cannot reproduce your bug locally. Same for @MANISHJOSHI23.

What I did:

```bash
manim-slides render example.py BasicExample ThreeDExample
manim-slides convert BasicExample ThreeDExample test.html --open
manim-slides convert BasicExample ThreeDExample test.html --open --use-template manim_slides/templates/revealjs.html
```

and all commands work as expected (both in full-screen and not).

FYI @MANISHJOSHI23, you might want to update your template file if it is too old.

**Comment by MANISHJOSHI23** [comment]
yes, I think the template file I was using was old, now it's working. Thank you  for your response and the great work you are doing
